regex-assembly/include/932130.ra

```diff
@@ -1,13 +1,13 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-\$\(.*\)
-\$\{.*\}
-<\(.*\)
->\(.*\)
-\$\(\(.*\)\)
+\$\(.*?\)
```

Comment:
While the change is probably fine, the matches will differ, depending on the context. Example:
For the string `$(some)thing)`, the original expression will match `$(some)thing)`, while the new expression will match `$(some)`.

Also, the expression would probably be more efficient when written as `\$\([^)]*?\)`, since in this form the wild card match can be performed in a single operation. It's probably overkill though, since the performance will mainly be an issue when the expression doesn't match.

Reply:
Thanks. Will give more time to think about this.